kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -24,9 +24,10 @@
 
         <div class="user-field">
           <label for="user-role"><span class="visuallyhidden">User Role</span></label>
-          <select v-model="role_new" id="user-role">
-            <option :selected="role_new == learner" v-if="role_new" value="learner"> Learner </option>
-            <option :selected="role_new == admin" value="admin"> Admin </option>
+          <select v-model="kind_new" id="user-role">
+            <option :value="LEARNER"> Learner </option>
+            <option :value="ADMIN"> Admin </option>
+            <option :value="COACH"> Coach </option>
```

Comment:
using `v-bind` isn't illegal, but it pretty blatantly goes against the patterns Vue is trying to enforce (`Declare initial values in the component's data option instead.`)